src/helpers/getAuthorNames.jsx

```diff
@@ -0,0 +1,15 @@
+import { mockAuthors } from '../data/data';
```

Comment:
Well, I would prefer to use `.js` extension instead of `jsx` since you have no any JSX-syntax here